maestrowf/datastructures/core/study.py

```diff
@@ -164,7 +187,8 @@ class Study(DAG, PickleInterface):
     """
 
     def __init__(self, name, description,
```

Comment:
This might be a knit-pick, but it might be better to move `draw=False` to the `configure` method. That's where `throttle` and other settings are set -- to me it makes a bit more sense to have it there.